BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -595,7 +595,9 @@ extension AttributedLabel {
                 }
                 label.insert(contentsOf: insertionString, at: insertionPoint)
             }
-            return label
+
+            // We need to replace all newlines with \n
+            return label.components(separatedBy: .newlines).filter { !$0.isEmpty }.joined(separator: " ")
```

Comment:
Tests please!

Reply:
To be explicit, theres no prohibition against new lines in accessibility labels. This is a style choice we've made.

Reply:
the tests were a good idea! you'll never believe this: they caught a bug (the change up above that i missed)

Reply:
Can we move the line into a private category on string in this file now that we have two copies?